pkg/container/vector/tools.go

```diff
@@ -296,5 +296,7 @@ func (v *Vector) extend(rows int, m *mheap.Mheap) error {
 	}
 	// Setup v.Col
 	v.setupColFromData(0, tgtSz/v.GetType().TypeSize())
+	// extend the null map
+	nulls.TryExpand(v.Nsp, tgtSz)
```

Comment:
Please remove the XXX comment on top.

Second, should you only expand when original v.Nsp is not empty?   I am not sure about this ...

Reply:
yeah, I removed the comments.

and nulls.TryExpand has do the check work ( if Nsp is nil or not)